samples/startup-script/index.js

```diff
@@ -59,8 +59,7 @@ async function createVM(name) {
   await operation.promise();
 
   console.log('Acquiring VM metadata...');
-  const [metadata] = await vm.getMetadata();
-  console.log(metadata);
+  const [{metadata}] = await vm.getMetadata();
```

Comment:
@stephenplusplus So, I don't know if this is right 🤷‍♂️   Is `vm.getMetadaa()` supposed to return `[Operation]`?  Or is it supposed to just directly return the metadata itself?